pkg/api/mlflow/dao/repositories/param.go

```diff
@@ -31,8 +32,50 @@ func NewParamRepository(db *gorm.DB) *ParamRepository {
 
 // CreateBatch creates []models.Param entities in batch.
 func (r ParamRepository) CreateBatch(ctx context.Context, batchSize int, params []models.Param) error {
-	if err := r.db.CreateInBatches(params, batchSize).Error; err != nil {
+	var err error
+	err = r.db.CreateInBatches(params, batchSize).Error
+	if err != nil {
+		// remove duplicate rows and try again
+		params, err1 := r.removeExactMatches(ctx, params)
```

Comment:
this is not good to use such names like `err2` or `err2` or any other.

Reply:
will rename